### ISSUE TITLE
Compile Error of ex_cldr 2.27.0

### ISSUE THREAD

**Issue by maennchen** [elixir-cldr/cldr#171]
Error:

```
==> ex_cldr
Compiling 1 file (.yrl)
Compiling 1 file (.xrl)
Compiling 2 files (.erl)
Compiling 43 files (.ex)

== Compilation error in file lib/cldr/config/rbnf_config.ex ==
** (ArgumentError) The environment variable $CLDR_PRODUCTION must be set to the
directory where the CLDR json data is stored.

See DEVELOPMENT.md for more information about CLDR data
and generating the json files.

    lib/cldr/config/config.ex:202: Cldr.Config.download_data_dir/0
    lib/cldr/config/rbnf_config.ex:9: (module)
could not compile dependency :ex_cldr, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile ex_cldr", update it with "mix deps.update ex_cldr" or clean it with "mix deps.clean ex_cldr"
Error: Process completed with exit code 1.
```

https://github.com/maennchen/athena/runs/5462301948?check_suite_focus=true

**Comment by kipcole9** [comment]
Fixed in [ex_cldr version 2.27.1](https://hex.pm/packages/ex_cldr/2.27.1). Also reported by @[dkln](https://github.com/dkln) in https://github.com/elixir-cldr/cldr/issues/170

Very sorry for the inconvenience, that was just sloppy on my part.

**Comment by maennchen** [comment]
@kipcole9 Oops, sorry for the duplicate.
